presentation/common/package.json

```diff
@@ -18,31 +18,47 @@
     "name": "Bentley Systems, Inc.",
     "url": "http://www.bentley.com"
   },
+  "type": "module",
   "main": "lib/cjs/presentation-common.js",
+  "types": "lib/cjs/presentation-common",
   "module": "lib/esm/presentation-common.js",
-  "typings": "lib/cjs/presentation-common",
+  "exports": {
+    ".": {
+      "import": "./lib/esm/presentation-common.js",
+      "require": "./lib/cjs/presentation-common.js"
+    },
+    "./locales/*": {
+      "import": "./lib/assets/locales/*",
+      "require": "./lib/assets/locales/*"
+    },
+    "./test-utils": {
+      "import": "./lib/esm/test/index.js",
+      "require": "./lib/cjs/test/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "scripts": {
     "build": "npm run -s ruleset-json-schema && npm run -s build:assets && npm run -s build:public && npm run -s build:cjs && npm run -s build:esm",
-    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:cjs": "internal-tools create-cjs-package-json -o ./lib/cjs && tsc -p tsconfig.cjs.json",
     "build:esm": "tsc -p tsconfig.esm.json",
-    "build:assets": "cpx \"./assets/**/*\" ./lib/cjs/assets && npm run -s pseudolocalize:assets",
+    "build:assets": "cpx \"./assets/**/*\" ./lib/assets && npm run -s pseudolocalize:assets",
```

Comment:
should we call this out in nextver.md?
do you think there will people who are trying to manually resolve your assets from here?

Reply:
I would be surprised if anyone is importing them besides `presentation-backend`. But added a section to be safe.